web/src/message_events.ts

```diff
@@ -261,13 +279,19 @@ export let update_views_filtered_on_message_property = (
     }
 };
 
-export function rewire_update_views_filtered_on_message_property(value) {
+export function rewire_update_views_filtered_on_message_property(
+    value: (message_ids: number[], property_term_type: string, property_value: boolean) => void,
```

Comment:
```suggestion
    value: typeof update_views_filtered_on_message_property,
```

(As a `rewire_` helper, this is only used by tests: #32296.)